library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -224,11 +264,24 @@ spdm_process_opaque_data_supported_version_data(IN spdm_context_t *spdm_context,
         return RETURN_UNSUPPORTED;
     }
     versions_list = (void *)(opaque_element_support_version + 1);
-    if ((versions_list[0] >> SPDM_VERSION_NUMBER_SHIFT_BIT) != SPDM_MESSAGE_VERSION_11) {
-        return RETURN_UNSUPPORTED;
+
+    spdm_secured_version_number_sort(versions_list, spdm_context->local_context.secured_message_version.spdm_version_count);
+    spdm_secured_version_number_sort(spdm_context->local_context.secured_message_version.spdm_version, spdm_context->local_context.secured_message_version.spdm_version_count);
+
+    for (secured_message_version_index = 0;
```

Comment:
Yes. I have updated the code.

Reply:
I do not see the update. How do you align the code?

The original code is problematic, because it compares the full version.
But only major/minor is required, patch/alpha should be ignored. See https://github.com/DMTF/libspdm/blob/main/library/spdm_requester_lib/libspdm_req_get_version.c#L109, spdm_get_version_from_version_number() function.